### PR TITLE
boolean fix and loading color editable

### PR DESCRIPTION
**Pull request by 00salmon**

### CODE REVIEW

File: packages/vanilla-components-grapesjs/package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
-  "name": "@saasquatch/vanilla-components-grapesjs",
-  "version": "1.0.11",
+  "name": "@kittenco/vanilla-components-grapesjs",
```
Comment (cbot83) [inline packages/vanilla-components-grapesjs/package.json]:
```suggestion
  "name": "@saasquatch/vanilla-components-grapesjs",
```

File: packages/vanilla-components-grapesjs/package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
...
+  "version": "1.0.33",
```
Comment (cbot83) [inline packages/vanilla-components-grapesjs/package.json]:
```suggestion
  "version": "1.0.12",
```

File: packages/vanilla-components/package.json [hunk 3]
```diff
@@ -1,6 +1,6 @@
 {
-  "name": "@saasquatch/vanilla-components",
-  "version": "1.0.3",
+  "name": "@kittenco/vanilla-components",
```
Comment (cbot83) [inline packages/vanilla-components/package.json]:
```suggestion
  "name": "@saasquatch/vanilla-components",
```

File: packages/vanilla-components/package.json [hunk 4]
```diff
@@ -1,6 +1,6 @@
...
+  "version": "1.0.21",
```
Comment (cbot83) [inline packages/vanilla-components/package.json]:
```suggestion
  "version": "1.0.4",
```

File: packages/vanilla-components/src/components/global-container/global-container.tsx [hunk 5]
```diff
@@ -7,19 +7,23 @@ import { css } from 'emotion';
...
   @Prop() fontfamily: string;
   @Prop() maxwidth: string;
-  @Prop() poweredby: boolean = true;
-
-  LoadingState() {
+  @Prop() poweredby: boolean;
+  
+  LoadingState(props) {
+    const style = css`
+      background-color: ${props.loadingcolor || '#439B76'};
```
Comment (cbot83) [inline packages/vanilla-components/src/components/global-container/global-container.tsx]: Should this default loading color go in contentful? I think this is the only prop that has a fall back default color in vanilla components. Is loading state a special case?

File: packages/vanilla-components/src/default-referred-widget-contentful.html [hunk 6]
```diff
@@ -7,29 +7,35 @@
   //
  -->
 
+
 <sqh-global-container 
   background="#ffffff"
```
Comment (cbot83) [inline packages/vanilla-components/src/default-referred-widget-contentful.html]: As per above, maybe loading state color should be set here.

File: packages/vanilla-components/src/components/global-container/global-container.tsx (hunk 5 above)
Comment (00salmon) [reply packages/vanilla-components/src/components/global-container/global-container.tsx]: I wasn't sure about backwards compatibility here. Current widgets won't have loadingcolor set, but then if vanilla components updates then they'd have loading color appear blank in the form.

Comment (jorgecgll) [reply packages/vanilla-components/src/components/global-container/global-container.tsx]: I like the idea of having defaults here, for when the user doesn't set values in the web components.

**Pull request merged**